### ISSUE TITLE
ISAICP-6541: Remove PHP 7.2 support. Support Drupal ^9.1

### ISSUE THREAD

**Issue by claudiu-cristea** [openeuropa/oe_bootstrap_theme#3]
Joinup ticket https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-6541

**Comment by pfrenssen** [comment]
I looked into this and it is not needed to increase the minimum PHP version to 7.3 to enable Drupal 9 support. Drupal 9 itself uses PHP 7.3 in core, but dependencies can still be compatible with older PHP versions, as long as their code works on PHP 7.3 (throws no notices etc).

I looked in the dependencies of core and there are dozens of libraries used that written in for older PHP versions, some are even as old as PHP 5.2.

So this means we can stay on PHP 7.2 without problems. I would even propose to lower it to PHP 7.1 so it is compatible with some older projects (such as Joinup) that are still stuck on this version for "reasons".

For this base theme the PHP side of things are not very complicated. There are no features in PHP 7.3 that would make this a requirement. Unless we really need `array_key_first()` or the fancy new HEREDOC syntax.
